helpers/config.py

```diff
@@ -293,17 +316,32 @@ def write_config(self):
         self.__config["date_modified"] = int(time.time())
 
         try:
-            with open(Config.CONFIG_FILE, "w") as f:
+            base_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+            config_file = "{}/{}".format(base_dir, Config.CONFIG_FILE)
+            with open(config_file, "w") as f:
                 f.write(json.dumps(self.__config))
 
-            os.chmod(Config.CONFIG_FILE, stat.S_IWRITE | stat.S_IREAD)
+            os.chmod(config_file, stat.S_IWRITE | stat.S_IREAD)
 
         except Exception as e:
             CLI.colored_print("Could not write configuration file", CLI.COLOR_ERROR)
             return False
```

Comment:
Does anything happen when `False` is returned? Should this `raise` after printing the error?

Reply:
Actually, don't need to raise an exception because we already print the error to the console.
`sys.exit()` should do the trick. 

At the beginning, I thought it should not be a blocker if config can't be saved to the disk that's why I chose to only return bool.